### PR TITLE
add links to expressions static values page

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-platform-docs/issues/2485.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [x] This change or page is part of a marketing blog, conference talk, or something else on a schedule. **CC**
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: versioned_docs/version-8.2/components/modeler/forms/form-element-library/forms-element-library-image.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ An element allowing the user to display images.
 
 ## Configurable properties
 
-- **Image source**: Specifies the image source via [expression](../../feel/language-guide/feel-expressions-introduction.md) or static value (hyperlink or data URI).
+- **Image source**: Specifies the image source via [expression](../../feel/language-guide/feel-expressions-introduction.md) or [static value](/docs/components/concepts/expressions.md) (hyperlink or data URI).
```
Comment (korthout) [inline versioned_docs/version-8.2/components/modeler/forms/form-element-library/forms-element-library-image.md:13]: 🤔 I'm not familiar with this part, I wonder whether this static value is as defined by the concepts. I hope someone else can comment on this one.

**Review by korthout — APPROVED** [review]
Thanks @christinaausley 🥇 

🔧 I was wondering why you've linked to the expressions page and not the [expressions vs static values](https://docs.camunda.io/docs/next/components/concepts/expressions/#expressions-vs-static-values) section of that page. I think it might be better to directly navigate readers to the spot where they can learn about static values. But I'm happy with these changes either way.

File: versioned_docs/version-8.2/components/modeler/forms/form-element-library/forms-element-library-image.md (hunk 1 above)
Comment (nikku) [reply versioned_docs/version-8.2/components/modeler/forms/form-element-library/forms-element-library-image.md:13]: @Skaiir Can you validate this is a solid addition?

**Comment by christinaausley** [comment]
> Thanks @christinaausley 🥇
> 
> 🔧 I was wondering why you've linked to the expressions page and not the [expressions vs static values](https://docs.camunda.io/docs/next/components/concepts/expressions/#expressions-vs-static-values) section of that page. I think it might be better to directly navigate readers to the spot where they can learn about static values. But I'm happy with these changes either way.

Adjusted this in my latest commit 🥳

**Review by korthout — APPROVED** [review]
Thanks @christinaausley 👍 LGTM

❓ @Skaiir let us know whether the linked static value definition is also valid for the `Image source` described in the form-element-library

**Comment by christinaausley** [comment]
Can open a subsequent PR to adjust based on response from @Skaiir -- merging this in 👍

**Pull request merged**